pom.xml

```diff
@@ -76,12 +78,12 @@
 		<dependency>
 			<groupId>org.thymeleaf.extras</groupId>

```

Comment:
Thymeleaf is not used in project. I think that there will be react as a rest api consumer/ frontend library

Reply:
It is used for Security.

Reply:
As far as i know it is used only when Thymeleaf template engine is used. Here, we have external, react library for UI layer 
and spring-boot-starter-security is enough.

Reply:
Thymeleaf is used for security, but only when you use template with it and you want to use some tags like: <div sec:authorize>. As we don't use thymeleaf at all you can just delete it.

Reply:
Ok, we can discuss security, but for now there is a class in code for which this dependecy is needed and is used (SecurityConfiguration.class). Removing this dependency from pom.xml will break the application build. I think that approach to security should be solved in other task.

Reply:
Application is broken anyway. There is a list in entity class without any mapping annotation so application is not starting correctly. Imo, just remove import, fix class that is using that import and close PR. We are just wasting time discusing about minor things.

Reply:
"Application is broken anyway", but it's not the explenation for adding new failures. "Fixing" class in this case means removing this class and I will not do this without knowledge of person, who added it. As I wrote before, I think it shouldn't be done here. Security is not minor thing.